### ISSUE TITLE
[TASK]: Add WYSIWYG Editor to Posts.Create

### ISSUE THREAD

**Issue by jacurtis** [DevMarketer/DevMarketer#49]
We need a good WYSIWYG editor on the posts.create page to create a new blog post.

**Comment by steve5287** [comment]
What do you think about CKeditor 5?

**Comment by steve5287** [comment]
Look at Pulse editor, is like WYSIWYG editor over a Markup editor, "works with react"

**Comment by zymawy** [comment]
Yeah Tinymce Could Be Useful With Vuejs [https://github.com/Eazymov/vue-mce](url)
